modules/ROOT/pages/monitoring/logging.adoc

```diff
@@ -375,6 +506,56 @@ There are also a couple of built-in templates available from the classpath, for
 | Same layout as the less flexible `<JsonLayout>`.
 |===
 
+==== Filters
+
+You can also configure filters to determine if and what log events are published and how.
+For example, you can filter out logs for queries executed by the `neo4j` user or for the `neo4j` database.
+
+[source,xml]
+----
+<RollingRandomAccessFile name="QueryLog" fileName="${config:server.directories.logs}/query.log"
+                                 filePattern="$${config:server.directories.logs}/query.log.%02i">
+            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss.SSSZ}{GMT+0} %-5p %m%n"/>
+            <Policies>
+                <SizeBasedTriggeringPolicy size="20 MB"/>
+            </Policies>
+            <!-- Filter out logs for queries executed by "neo4j" user -->
+            <Filters>
```

Comment:
if someone will do that on the log level - that's not really nice no one will ever see those queries over there and any support case maintainer will gonna be puzzled, so not sure we need to advertise this feature

Reply:
Could you be able to help me with some examples of using Filters and Plugins?

Reply:
So for example `Neo4jDebugLogLayout` that is mentioned in the docs is in fact `plugin`. 

So filter and plugins are more advanced things that by default neo4j users should not update/touch. Was there a specific request why we need to include them? My thinking is that if you are already at the stage where you want to write your own plugin or use a custom filter - you would be able to find that in log4j docs/examples.